DFC.ServiceTaxonomy.GraphSync/GraphSyncers/Helpers/EmbeddedContentItemsGraphSyncer.cs

```diff
@@ -441,5 +441,24 @@ private ContentItem[] ConvertToContentItems(JArray? contentItems)
                 throw new GraphSyncException("Embedded content container does not contain ContentItems.");
             return embeddedContentItems;
         }
+
+        public async Task AddRelationship(IDescribeRelationshipsContext context)
+        {
+            var items = (JArray)context.ContentItem.Content["FlowPart"]["Widgets"];
```

Comment:
@lazcool Is there a better way to get ["FlowPart"]["Widgets"]? I'm pretty sure there is